### PR TITLE
Unorganized mess

### PR DESCRIPTION
**Pull request by StachuDotNet**

### CODE REVIEW

File: backend/tests/TestUtils/TestFileParser.fs [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+                                        _),
+                         _,
+                         _,
+                         _) -> Some(p1, p2, p3)
+        | _ -> None
+      else
+        None)
+    |> List.tryHead
+
+  let rec parseModule
```
Comment (pbiggar) [inline backend/tests/TestUtils/TestFileParser.fs:101]: parseModule can be used for parsing whole files - it's not just for tests.

File: packages/openai/main.dark [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+    type CreateResponse =
+        { id: string
+          object: string
+          Created: int
+          Model: string
+          Choices: Choice[]
+          Usage: Usage }
+
+    let chat (config: Config) : ConfigWithChatContext =
+        ConfigWithChatContext(
```
Comment (pbiggar) [inline packages/openai/main.dark:188]: What am I looking at here?

File: packages/openai/main.dark [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+        )
+
+    let create (request: CreateRequest) (config: ConfigWithEditContext) : CreateResponse =
+        config.HttpRequester.postRequest<CreateRequest, CreateResponse> config.ApiConfig request
+
+
+(* OpenAI *)
+type OpenAIComputed() =
+    // Required - creates default "starting" values
+    member _.Yield _ =
```
Comment (pbiggar) [inline packages/openai/main.dark:235]: What are members in Dark?

File: packages/openai/main.dark (hunk 2 above)
Comment (StachuDotNet) [reply packages/openai/main.dark:188]: A totally-unfinished copy-and-paste from `openai-fsharp` that I was in-progress of massaging to be parse-able.
(so the answer around "members" below is that "I didn't port that code yet")

**Pull request closed without merging**